src/commands/execute.spec.ts

```diff
@@ -0,0 +1,118 @@
+/* eslint no-implicit-dependencies: "off" */
```

Comment:
We don't have eslint in this project 🤔

Reply:
Just added in case of migrating to eslint. AFAIR tslint is (or soon will be) deprecated